cime_config/testdefs/testmods_dirs/cam/def_cmcc/shell_commands

```diff
@@ -0,0 +1,9 @@
+./xmlchange CAM_CONFIG_OPTS='-phys cam_dev -chem waccm_sc_mam4 -nlev 83'
+./xmlchange SSTICE_MESH_FILENAME='$DIN_LOC_ROOT/share/meshes/fv0.47x0.63_141008_FIX102722_polemod_ESMFMesh.nc'
+./xmlchange SSTICE_DATA_FILENAME='$DIN_LOC_ROOT/atm/cam/sst/sst_HadOIBl_bc_0.47x0.63_1850_2017_c180507.nc'
+./xmlchange SSTICE_YEAR_ALIGN=1980
+./xmlchange SSTICE_YEAR_START=1980
+./xmlchange MOSART_SIM_YEAR=1980
+./xmlchange RUN_STARTDATE=1980-01-01
+./xmlchange CLM_FORCE_COLDSTART=on
+./xmlchange NTASKS=864
```

Comment:
I would discourage the usage of NTASKS settings that are machine specific here. This belongs under config_pes.xml in CAM's cime_config directory.

Reply:
Thanks, I will move NTASKS to the config_pes.